DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/VPNUIActionHandler.swift

```diff
@@ -40,7 +40,21 @@ final class VPNUIActionHandler: VPNUIActionHandling {
         self.proxySettings = proxySettings
     }
 
-    public func moveAppToApplications() async {
+    func askUserToReportIssues(withDomain domain: String) async {
```

Comment:
This is new - it was previously handled elsewhere but it's more appropriate here.